randovania/data/json_data/prime3.txt

```diff
@@ -3812,12 +4678,12 @@ Asset id: 1808913861159744919
 Command Courtyard
 Asset id: 18221971457754325372
 > Dock to Courtyard Passage; Heals? False
-  * Normal Door to Courtyard Passage/Dock to Command Courtyard
+  * Plasma Door to Courtyard Passage/Dock to Command Courtyard
   > Bottom Floor
       Trivial
```

Comment:
This is not a Plasma Door, its a normal door.

Connect this node to Lift Hub door with requirement "After Barrier Event"